### PR TITLE
figmas

### PR DESCRIPTION
**Pull request by MaxymKoz**

### CODE REVIEW

File: index.html.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="https://cdnjs.com/libraries/normalizenormalize.css" integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
+        <link rel="stylesheet" href="my.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&display=swap&subset=cyrillic,cyrillic-ext,latin-ext" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
+        <link rel=" preconnect " href="https://fonts.googleapis.com ">
```
Comment (Ilyaobr) [inline index.html.html:12]: Try to remove

File: index.html.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <link rel="preconnect " href="https://fonts.gstatic.com " crossorigin>
```
Comment (Ilyaobr) [inline index.html.html:13]: Try to remove

File: index.html.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&family=Roboto:wght@100&family=Space+Mono&display=swap " rel="stylesheet ">
+    </head>
+
+    <body>
```
Comment (Ilyaobr) [inline index.html.html:17]: Add class name to "body"

File: index.html.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<DOCTYPE html>
```
Comment (velmyk) [inline index.html.html:1]: Hello

File: index.html.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
+        <link rel=" preconnect " href="https://fonts.googleapis.com ">
+        <link rel="preconnect " href="https://fonts.gstatic.com " crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100&family=Roboto:wght@100&family=Space+Mono&display=swap " rel="stylesheet ">
+    </head>
+
+    <body>
+        <div class="main block1">
```
Comment (Ilyaobr) [inline index.html.html:18]: Change all class names

File: my.css [hunk 6]
```diff
@@ -0,0 +1,338 @@
+* {
+    display: flex;
+}
+
+body {
```
Comment (Ilyaobr) [inline my.css:5]: Don't set properties via tag name

File: my.css [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+
+.l01 {
+    color: white;
+    position: absolute;
+    flex-direction: column;
+    width: 380px;
+    margin-top: 351px;
+}
+
+li {
```
Comment (Ilyaobr) [inline my.css:71]: Don't set properties via tag name

File: my.css [hunk 8]
```diff
@@ -0,0 +1,338 @@
...
+.list3>div {
+    padding: 16 21 16 59;
+}
+
+.button3 {
+    background-color: #EC632A;
+    margin: 690 113.37 40 112;
+}
+
+button {
```
Comment (Ilyaobr) [inline my.css:332]: Don't set properties via tag name

File: my.css [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+
+.l03 {
+    color: white;
+    position: absolute;
+    flex-direction: column;
+    width: 380px;
+    margin-top: 351px;
+}
+
+.list3>div {
```
Comment (Ilyaobr) [inline my.css:323]: It is not advisable to do

File: my.css [hunk 10]
```diff
@@ -0,0 +1,338 @@
...
+    width: 380px;
+    margin-top: 351px;
+}
+
+.lmem {
+    width: 380px;
+    height: 50px;
+}
+
+.list2>div {
```
Comment (Ilyaobr) [inline my.css:262]: It is not advisable to do

File: index.html.html [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+            </div>
+            <h1 class="t02"> Business</h1>
+            <p class="tp002">Website
+
+                <div class="price2"><span class="price02">299€</span></div>
+
+                <div class="describe2"><span class="spbt">Advanced website made in the world's most popular CMS system, Wordpress.</span></div>
+
+                <ol class="l02">
+                    <li class="lmem"><img class="pictures" src="./Shape2.png ">Website in WordPress</li>
```
Comment (Ilyaobr) [inline index.html.html:64]: "CheckBoxBlue" class name is better

File: index.html.html [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+                    <li class="lmem"><img class="pictures " src="./Shape2.png ">Theme customized</li>
+
+                    <li class="lmem"><img class="pictures " src="./Shape2.png ">Help and guidance</li>
+
+                    <li class="lmem"><img class="pictures " src="./Shape2.png ">System introduction</li>
+
+                    <li class="lmem"><img class="pictures " src="./Shape2.png ">1 month free support</li>
+
+                </ol>
+                <button class="button2 "><span class="spanbutton1 "> ORDER NOW</button>
```
Comment (Ilyaobr) [inline index.html.html:77]: remove space in class name

File: my.css (hunk 6 above)
Comment (Ilyaobr) [inline my.css:5]: use flexbox to position at the center of the page

File: my.css [hunk 13]
```diff
@@ -0,0 +1,338 @@
...
+    background-color: #5BB189;
+    margin-top: 130px;
+    margin-bottom: 535px;
+}
+
+.price01 {
+    color: white;
+    margin: 19 110 19 150;
+    position: absolute;
+    font-family: 'Montserrat';
```
Comment (Ilyaobr) [inline my.css:44]: font-family: Montserrat, 'sans-serif';

File: index.html.html [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+
+                <li class="lmem"><img class="pictures " src="./Shape3.png "> Theme customized</li>
+
+                <li class="lmem"><img class="pictures " src="./Shape3.png "> Help and guidance</li>
+
+                <li class="lmem"><img class="pictures " src="./Shape3.png "> System introduction</li>
+
+                <li class="lmem"><img class="pictures " src="./Shape3.png "> 2 weeks free support</li>
+            </ol>
+            <button class="button3 "><span class="spanbutton1 "> ORDER NOW
```
Comment (velmyk) [inline index.html.html:101]: I don't see a closing tag for the `span`.
Try to remove `span`

File: my.css [hunk 15]
```diff
@@ -0,0 +1,338 @@
...
+    height: 105px;
+    background-color: #5BB189;
+    margin-top: 130px;
+    margin-bottom: 535px;
+}
+
+.price01 {
+    color: white;
+    margin: 19 110 19 150;
+    position: absolute;
```
Comment (velmyk) [inline my.css:43]: This element should not be `absolute`

File: my.css [hunk 16]
```diff
@@ -0,0 +1,338 @@
...
+    letter-spacing: 0px;
+    text-align: center;
+}
+
+.describe1 {
+    text-align: center;
+    width: 327.49px;
+    height: 36px;
+    position: absolute;
+    margin: 275 26.51 459 26;
```
Comment (velmyk) [inline my.css:59]: Looks like copy-paste from `figma`.
It should not have such margins

File: my.css [hunk 17]
```diff
@@ -0,0 +1,338 @@
...
+.spanbutton1 {
+    position: absolute;
+    font-family: 'Montserrat';
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 17px;
+    letter-spacing: 0px;
+    height: 17px;
+    width: 99.91453552246094px;
```
Comment (velmyk) [inline my.css:98]: 🥴

**Review by velmyk — CHANGES_REQUESTED** [review]

**Pull request merged**